core/src/test/java/org/elasticsearch/action/admin/cluster/node/tasks/TasksIT.java

```diff
@@ -145,6 +145,15 @@ public void testTaskCounts() {
         assertThat(response.getTasks().size(), greaterThanOrEqualTo(cluster().numDataNodes()));
     }
 
+    public void testTaskInfoToString() {
+        // Run only on data nodes
+        ListTasksResponse response = client().admin().cluster().prepareListTasks().setActions(ListTasksAction.NAME).get();
+        assertThat(response.getTasks().size(), equalTo(1));
+        String taskInfoString = response.getTasks().get(0).toString();
+        assertThat(taskInfoString, startsWith("{"));
+        assertThat(taskInfoString, endsWith("}"));
```

Comment:
I think that if the toString didn't throw error we are already good. Otherwise we could really check its content. I find it a bit odd to only check that it's proper json.